### ISSUE TITLE
HTC / HTC++

### ISSUE THREAD

**Issue by vadimkantorov** [facebookresearch/detectron2#4379]
Hi @lyttonhao @HannaMao!

"Exploring Plain Vision Transformer Backbones for Object Detection" (https://arxiv.org/abs/2203.16527) mentions experiments with HTC/HTC++ detector architectures for https://github.com/facebookresearch/detectron2/tree/main/projects/ViTDet. Are these implemented anywhere in detectron2?

Thanks!

**Comment by lyttonhao** [comment]
Hi, we don't use HTC/HTC++ detectors in the papers of MViTv2 or ViTDet.

**Comment by vadimkantorov** [comment]
You are right, sorry! I'll repurpose this issue as a feature request then!
